sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -54,7 +54,7 @@ statement
         (TBLPROPERTIES tablePropertyList)?
         (AS? query)?                                                   #createTable
     | ANALYZE TABLE tableIdentifier partitionSpec? COMPUTE STATISTICS
-        (identifier | FOR COLUMNS identifierSeq?)                      #analyze
+        (identifier | FOR COLUMNS identifierSeq?)?                     #analyze
```

Comment:
I had to update the `StatisticsSuite` and change its parser to the `HiveSqlParser` & this error popped up. I could also do this in a separate PR.